fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -36,12 +36,23 @@ class ConditionalSelectionRate(ClassificationMoment):
     where error(A = a) = total_error
     """
 
-    def __init__(self, ratio=1.0):
+    def __init__(self, difference_bound=None, ratio_bound_slack=None, ratio_bound=None):
         """Initialize with the ratio value."""
         super(ConditionalSelectionRate, self).__init__()
-        if not (0 < ratio <= 1):
-            raise ValueError(_MESSAGE_RATIO_NOT_IN_RANGE)
-        self.ratio = ratio
+        if (difference_bound is None) and (ratio_bound_slack is None):
+            raise ValueError("One of difference_bound and ratio_bound_slack is required.")
```

Comment:
Please could these strings be made into constants, like the one in the ratio check.

Reply:
Yep, do you want the strings as constant in the same file as the ratio check?

Reply:
To be consistent, that would be good.